### PR TITLE
adding Milestones

### PR DESCRIPTION
**Pull request by Arshia01**
I am adding Milestones 1 to 5.

### CODE REVIEW

File: Arshia/Milestone-2/Task1/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            <h4>Father</h4>
+            <label for = 'ffname'>First Name:</label>
+            <input type = 'text' id = 'ffname'>
+            <label for = 'fmname'>Middle Name:</label>
+            <input type = 'text' id = 'fmname'>
+            <label for = 'flname'>Last Name:</label>
+            <input type = 'text' id = 'fLname'>
+            <br>
+            <br>
+            Occupation:&nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp&nbsp&nbsp&nbsp
```
Comment (Manvityagi) [inline Arshia/Milestone-2/Task1/index.html:49]: Using nbsp is not a good practice

File: Arshia/Milestone-3/index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            My Projects</i></h2>
+
+          &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp 
+          &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp
+          &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp
+          &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp
+          &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp
+            <img src = './Images/p1.jpg'>
+            <h3>&nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp 
+                &nbsp  &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp &nbsp
```
Comment (Manvityagi) [inline Arshia/Milestone-3/index.html:29]: Oops! Lots of nbsp. Ask your mentors on the group a good way to deal with this requirement.

**Review by Manvityagi — COMMENTED** [review]
The rest looks fine!! Good Luck with the next milestones :)

**Comment by Arshia01** [comment]
Okayy Thanks !

File: Arshia/Milestone-3/index.html (hunk 2 above)
Comment (Arshia01) [reply Arshia/Milestone-3/index.html:29]: Do I correct it and send it again or is it okay?

**Pull request merged**